src/neuroconv/datainterfaces/ecephys/phy/phydatainterface.py

```diff
@@ -41,3 +62,29 @@ def __init__(
         verbose : bool, default: True
         """
         super().__init__(folder_path=folder_path, exclude_cluster_groups=exclude_cluster_groups, verbose=verbose)
+
+    def add_to_nwbfile(
+        self,
+        nwbfile: NWBFile,
+        metadata: Optional[DeepDict] = None,
+        stub_test: bool = False,
+        write_ecephys_metadata: bool = False,
+        write_as: Literal["units", "processing"] = "units",
+        units_name: str = "units",
+        units_description: str = "Autogenerated by neuroconv.",
+    ):
+
+        super().add_to_nwbfile(
+            nwbfile=nwbfile,
+            metadata=metadata,
+            stub_test=stub_test,
+            write_ecephys_metadata=write_ecephys_metadata,
+            write_as=write_as,
+            units_name=units_name,
+            units_description=units_description,
+        )
+
+        max_channel = self.get_max_channel()
+        nwbfile.units.add_column(name="max_channel", description="Channel with maximum amplitude", data=max_channel)
```

Comment:
I think that you should add the property "max_channel"  to the sorting object. That way the function `add_units` will take care of edge cases like:
* The units table already exists and the size of max_channel does not match the size.
* The property 'max_channel" already exists.

Reply:
Yes, that's what I tried to do first, but then I ran into the problem that the integers were automatically converted to floats: #960

Reply:
One hack around if you want to get this going before we settle things in #960 is to check if the intended table (either the units of the one provided by name) already exists. In the case that it doesn not and this is the first pass you can safely add your lines in the bottom.

Reply:
Ok, marking this as resolved.